networkit/cpp/graph/KruskalMSF.cpp

```diff
@@ -32,7 +32,7 @@ void KruskalMSF::run() {
         // sort edges in decreasing weight order
         std::vector<MyEdge> sortedEdges; // (m);
         std::transform(G->edgeWeightRange().begin(), G->edgeWeightRange().end(),
-                       sortedEdges.begin(), [](const auto &edge) -> MyEdge {
+                       std::back_inserter(sortedEdges), [](const auto &edge) -> MyEdge {
```

Comment:
Since transform should work on already allocated memory, it might be beneficial performance-wise to pre-allocate the vector on definition.

Reply:
Makes a lot of sense. Will take care of this.